client/src/app/site/users/components/user-list/user-list.component.html

```diff
@@ -12,12 +21,30 @@
     <!-- prefix column -->
     <ng-container matColumnDef="group">
         <mat-header-cell *matHeaderCellDef mat-sort-header> Group </mat-header-cell>
-        <mat-cell *matCellDef="let user"> {{user.groups}} {{user.structureLevel}} </mat-cell>
+        <mat-cell *matCellDef="let user">
+            <div class='groupsCell'>
+                <span *ngIf="user.groups">
```

Comment:
This is true, if `user.groups` is `[]`. Please check for the length of the array.